google-cloud-clients/google-cloud-automl/pom.xml

```diff
@@ -88,4 +75,4 @@
       </plugin>
     </plugins>
   </reporting>
-</project>
+</project>
```

Comment:
Same here

Reply:
already add empty new line just below the last line.